.eslintrc.js

```diff
@@ -30,6 +30,13 @@ module.exports = {
     // when they're not:
     "no-unused-vars": "off",
     "@typescript-eslint/no-floating-promises": "error",
-    "license-header/header": [process.env.CI ? "error" : "warn", "./resources/license-header.js"],
+    "license-header/header": [
+      process.env.CI ? "error" : "warn",
+      "./resources/license-header.js",
+    ],
   },
+  // ESLint will try to parse these files using TypeScript,
+  // but since they're not part of the project, it will complain
+  // about them not being listed in tsconfig.json:
```

Comment:
This is because our eslint config is wrong, it should only apply the ts linting when the file extension is correct.
We should discuss at some point fixing our configs.

There is an `overrides` directive in eslint that can be file extensions based and where you can specify `parserOptions` and `plugins`! :)

I'd say let's do this separately though.